pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -204,9 +202,15 @@ in lib.makeScopeWithSplicing
       "TSORT=cat"
       # Can't process man pages yet
       "MKSHARE=no"
+    ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # GNU objcopy produces broken .a libs which won't link into dependers.
+      # Makefiles only invoke `$OBJCOPY -x/-X`, so cctools strip works here.
+      "OBJCOPY=${buildPackages.darwin.cctools}/bin/strip"
     ];
     RENAME = "-D";
 
+    passthru.tests = { netbsd-install = self.install; };
```

Comment:
I would have perhaps added a comment as to why this test is added. Might be hard to figure out what this is testing exactly in the future.